examples/fullstack-e2e-encrypted-locker-nextjs/app/private/LogoutButton.tsx

```diff
@@ -0,0 +1,20 @@
+"use client";
+
+import { useRouter } from "next/navigation";
+import { removeLoginKeys } from "../utils/auth";
+
+export default function LogoutButton() {
+  const router = useRouter();
+  return (
+    <button
+      className="text-blue-500"
+      onClick={async () => {
+        await fetch("/api/logout", { method: "POST" });
+        removeLoginKeys();
```

Comment:
@steos a try/catch with `removeLoginKeys` in a finally block would be nice. The session will expire anyway, but a user that can't remove the keys from the sessionStorage because the server is down is not ideal :)